core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserStoreManager.java

```diff
@@ -491,6 +490,22 @@ protected abstract void doUpdateRoleName(String roleName, String newRoleName)
     protected abstract String[] doListUsers(String filter, int maxItemLimit)
             throws UserStoreException;
 
+    /**
+     * Count claims.
+     *
+     * @param claimURI Claim uri
+     * @param valueFilter Filter
+     * @return null.
+     * @throws UserStoreException UserStoreException
+     */
+    public Long doCountClaims(String claimURI, String valueFilter) throws UserStoreException{
+
+        if (log.isDebugEnabled()) {
+            log.debug("Operation is not supported in: " + this.getClass());
+        }
+        return null;
```

Comment:
Let's throw an exception saying, this operation is not supported rather than returning null.